stellar_sdk/client/requests_client.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         backoff_factor: float = DEFAULT_BACKOFF_FACTOR,
         session: Session = None,
         stream_session: Session = None,
+        custom_headers: Dict[str, str] = {},
```

Comment:
I personally tend to modify it as follows, although this probably won't cause a bug in the current situation, for more information: https://stackoverflow.com/a/26320938

```suggestion
        custom_headers: Dict[str, str] = None,
```